app/serializers/quiz_serializer.rb

```diff
@@ -15,5 +15,8 @@
 #  index_quizzes_on_owner_id  (owner_id)
 #
 class QuizSerializer < ApplicationSerializer
-  attributes :hashid, :owner_id
+  attributes :owner_id
+
+  typelize 'string'
+  attribute(:hashid, &:hashid)
```

Comment:
It's a tiny bummer that we need to break this out from `attributes` to type it, but really not that bad.